server/middlewares/validations/UserValidation.js

```diff
@@ -0,0 +1,53 @@
+import Validator from 'validatorjs';
+
+class UserValidation {
+  /**
+   * @description Validates the request payload to create a user
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @param {Object} next The next middleware
+   * @returns The next middleware to handle the user signup
+   */
+  static signupValidation(req, res, next) {
+    const userProperties = {
+      firstName: 'required|alpha|min:2|max:15',
+      lastName: 'required|alpha|min:2|max:15',
+      email: 'required|email',
+      username: 'required|alpha_num|min:5|max:15',
+      password: 'required|alpha_num|min:8|max:20',
+      confirmPassword: 'required|alpha_num|min:8|max:20|same:password',
+    };
+
+    const validator = new Validator(req.body, userProperties);
+    validator.passes(() => {
+      if (UserValidation.passwordCheck(req.body.password)) return next();
+      return res.status(400).json({
+        status: 'error',
+        errors: {
+          password: ['Password most contain an Upper case letter, a lower case letter and a number.'],
```

Comment:
Hi Chima, I see you have made changes to some of the vocabulary errors, however kindly change the 'most' here to 'must'.
Thanks.